### PR TITLE
Authentication flow

### PR DESCRIPTION
**Pull request by Dhiraj1405**

### CODE REVIEW

**Comment by Dhiraj1405** [comment]
Closes #24

File: API authentication.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# API authentication 
+
+## Overview 
+
+Verfies the identity of the user. To make transactions on HotWax’s platform secure, all apps connecting with HotWax API might need to authenticate when making API requests.
```
Comment (dt2patel) [inline API authentication.md]:
```suggestion
Verfies the identity of the user. To make transactions on HotWax’s platform secure, all apps connecting with HotWax API need authentication when making API requests.
```

File: API authentication.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Authentication Method Used
+
+### Bearer token authentication
```
Comment (dt2patel) [inline API authentication.md:9]: we have a video about generating and retrieving bearer tokens, maybe we can use that here?

File: Inventory/Update Inventory.md [hunk 3]
```diff
@@ -18,8 +18,16 @@ Updated inventory in HotWax: 8
...
+
 Content-Type: application/json
 
+#### Authentication
+
+To access this endpoint, you need to include the authorization token in the request header.
+
+`Authorization: Bearer <access_token>`
+
+To know more about Bearer token authentication, read [this](https://github.com/Dhiraj1405/oms-documentation/blob/Authentication-flow/API%20authentication.md)
```
Comment (dt2patel) [inline Inventory/Update Inventory.md]:
```suggestion
To learn more about Bearer token authentication, read [this](https://github.com/Dhiraj1405/oms-documentation/blob/Authentication-flow/API%20authentication.md)
```

**Review by dt2patel — CHANGES_REQUESTED** [review]
Minor suggestions

File: API authentication.md (hunk 2 above)
Comment (Dhiraj1405) [reply API authentication.md:9]: okay I am adding it here

**Comment by Dhiraj1405** [comment]
Accepeted suggestions and add the HotWax Video below. Please merge @dt2patel

File: Inventory/Update Inventory.md [hunk 4]
```diff
@@ -18,8 +18,16 @@ Updated inventory in HotWax: 8
...
+
 Content-Type: application/json
 
+#### Authentication
+
+To access this endpoint, you need to include the authorization token in the request header.
+
+`Authorization: Bearer <access_token>`
+
+To learn more about Bearer token authentication, read [this](https://github.com/Dhiraj1405/oms-documentation/blob/Authentication-flow/API%20authentication.md)
```
Comment (dt2patel) [inline Inventory/Update Inventory.md:30]: remember to update this link to OMS repo link when this doc is published

Comment (Dhiraj1405) [reply Inventory/Update Inventory.md:30]: sure noted

**Pull request merged**